includes/classes/Blocks.php

```diff
@@ -0,0 +1,89 @@
+<?php
+/**
+ * Jobber Blocks.
+ *
+ * @package Jobber
+ */
+
+namespace Jobber;
+
+use TenupFramework\Module;
+
+/**
+ * Base class for Jobber Blocks
+ */
+class Blocks {
+	use Module;
+
+	/**
+	 * Can we register this module?
+	 *
+	 * @return bool
+	 */
+	public function can_register(): bool {
+		return true;
+	}
+
+	/**
+	 * Hook the module into WP.
+	 */
+	public function register() {
+		add_action( 'init', [ $this, 'register_block_types' ] );
+	}
+
+	/**
+	 * Register the block types.
+	 */
+	public function register_block_types() {
+		register_block_type(
+			JOBBER_PLUGIN_PATH . 'blocks/jobber',
+			[
+				'render_callback' => [ $this, 'render_block' ],
+			]
+		);
+	}
+
+	/**
+	 * Render the block.
+	 *
+	 * @param array $attributes The block attributes.
+	 * @return string
+	 */
+	public function render_block( array $attributes ): string {
+		$form_type = ! empty( $attributes['formType'] ) ? sanitize_text_field( $attributes['formType'] ) : 'request';
+
+		$jobber   = new \Jobber\Jobber();
+		$response = $jobber->get_form( $form_type );
+
+		if ( is_wp_error( $response ) ) {
```

Comment:
If we encounter an error or we don't have a valid URL to display, wondering if we want to output an error message. I think that would just confuse users and may be better to just not render anything